installer/bundle/bundle_skel.sh

```diff
@@ -553,16 +552,6 @@ do
             shift 1
             ;;
 
-        --collectd)
```

Comment:
I don't think this is necessary as I expect this message to be a part of the release notes.